src/components/pages/Theme/Theme.jsx

```diff
@@ -261,6 +262,42 @@ export const Theme = () => {
         </Segment>
       </div>
 
+      <div className='section'>
+        <Segment padded>
+          <Header as='h2' dividing>
+            Chip
+          </Header>
+          <div className='row'>
+            <Label as='a' color='green'>
+              Filled Green
+              <Icon name='delete' />
+            </Label>
+            <Label as='a' color='blue'>
```

Comment:
I think the interactive labels should use `as="button"`. We'll just need to add a new selector with `button`.

```
a.ui.label.green:hover,
button.ui.label.green:hover {...}
```

Let's also ensure the `a` and `button` use `cursor: pointer`.

```
a.ui.label,
button.ui.label {
  cursor: pointer;
}
```